src/main/ApplicationImpl.cpp

```diff
@@ -433,6 +434,13 @@ ApplicationImpl::getJsonInfo()
     info["ledger"]["baseFee"] = lcl.header.baseFee;
     info["ledger"]["baseReserve"] = lcl.header.baseReserve;
     info["ledger"]["maxTxSetSize"] = lcl.header.maxTxSetSize;
+
+    auto currentHeaderFlags = LedgerHeaderUtils::getFlags(lcl.header);
+    if (currentHeaderFlags != 0)
```

Comment:
I'd say there are two reasons - 
1. We don't want to show the flags field before it can be set (we could accomplish this by checking the ledger version).
2. No flags set isn't interesting and is the most likely scenarion, so no need to pollute the info endpoint.

Reply:
Reasonable enough.